### PR TITLE
zh-translation: /docs/setup/platform-setup/minikube/index.md

### PR DESCRIPTION
**Pull request by luxious**
ref: https://github.com/servicemesher/istio-official-translation/issues/1297

### CODE REVIEW

File: content/zh/docs/setup/platform-setup/minikube/index.md [hunk 1]
```diff
@@ -1,55 +1,50 @@
 ---
 title: Minikube
-description: Instructions to setup minikube for Istio.
+description: 在 Minikube 上配置 Istio.
```
Comment (rootsongjc) [inline content/zh/docs/setup/platform-setup/minikube/index.md]: 句尾请用中文句号。

Comment (luxious) [reply content/zh/docs/setup/platform-setup/minikube/index.md]: solved

**Comment by luxious** [comment]
> @luxious: The following test **failed**, say `/retest` to rerun them all:
> Test name 	Commit 	Details 	Rerun command
> lint_istio.io 	[25637e2](https://github.com/istio/istio.io/commit/25637e26d0a7ef25265d0bd38257fb319a25edb2) 	[link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio.io/5404/lint_istio.io/1184) 	`/test lint_istio.io`

/retest

**Pull request merged**